io/plugins/eu.esdihumboldt.hale.io.gml.ui/src/eu/esdihumboldt/hale/io/gml/ui/GmlWriterSettingsPage.java

```diff
@@ -99,9 +112,44 @@ protected void createContent(Composite page) {
 		// default
 		nilReason.setSelection(true);
 
+		Group writeFormat = new Group(page, SWT.NONE);
+		writeFormat.setLayout(GridLayoutFactory.swtDefaults().numColumns(2).create());
+		writeFormat.setText("Formatted number output for geometry");
+		groupData.applyTo(writeFormat);
+
+		Label lbl = new Label(writeFormat, SWT.NONE);
+		lbl.setLayoutData(GridDataFactory.swtDefaults().align(SWT.END, SWT.CENTER).create());
+		lbl.setText("Format: ");
+
+		formattedText = new Text(writeFormat, SWT.SINGLE | SWT.BORDER);
```

Comment:
Instead of having the empty text field implicitly setting no format I would prefer if there was a checkbox that activates having a format. The format field can then have a sensible default from the start.